src/app/CommandHandlerExchangeInterface.h

```diff
@@ -102,6 +102,18 @@ class CommandHandlerExchangeInterface
      * Called by CommandHandler to relay this information to the requester.
      */
     virtual void ResponseDropped() = 0;
+
+    /**
+     * @brief Gets the maximum size of a packet buffer to encode a Command
+     * message. This size depends on the underlying session used by the
```

Comment:
This is encoding a command response message, no?

Reply:
Yes. modified comment.